src/main/java/com/commercetools/sync/commons/models/GraphQlBaseRequest.java

```diff
@@ -0,0 +1,22 @@
+package com.commercetools.sync.commons.models;
+
+import io.sphere.sdk.client.SphereRequest;
+
+public interface GraphQlBaseRequest<T> extends SphereRequest<T>
```

Comment:
Can you describe what T should be?

Reply:
```suggestion
public interface GraphQlBaseRequest<T extends GraphQlBaseResult<?>> extends SphereRequest<T>
```

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/649/commits/d87506aff7ee6ab970286c0b42ba26ca2ab9c0f9